### ISSUE TITLE
Exception caused by update from PyCharm 2019.3.3 to 2020.1

### ISSUE THREAD

**Issue by tarasivashchuk** [ChrisRM/material-theme-jetbrains#1553]
I am running OSX 10.13 High Sierra with PyCharm Professional Edition. Error started showing up with update to 2020.1, and I used the plugin fine before. After this exception, the material theme does not load properly with almost nothing updated (maybe nothing, unsure as switched theme also?).

Let me know if there's anything else I can do to help!

@mallowigi - tagging you for notification

Here is the exception:
<pre style='color:#000000;background:#ffffff;'>java<span style='color:#808030; '>.</span>lang<span style='color:#808030; '>.</span>NullPointerException
at java<span style='color:#808030; '>.</span>base<span style='color:#808030; '>/</span>java<span style='color:#808030; '>.</span>util<span style='color:#808030; '>.</span>Objects<span style='color:#808030; '>.</span>requireNonNull<span style='color:#808030; '>(</span>Objects<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>221</span><span style='color:#808030; '>)</span>
at com<span style='color:#808030; '>.</span>mallowigi<span style='color:#808030; '>.</span>idea<span style='color:#808030; '>.</span>MTBundledThemesManager<span style='color:#808030; '>.</span>loadBundledThemes<span style='color:#808030; '>(</span>MTBundledThemesManager<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>97</span><span style='color:#808030; '>)</span>
at com<span style='color:#808030; '>.</span>mallowigi<span style='color:#808030; '>.</span>idea<span style='color:#808030; '>.</span>MTLafComponent<span style='color:#808030; '>.</span>initComponent<span style='color:#808030; '>(</span>MTLafComponent<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>111</span><span style='color:#808030; '>)</span>
at com<span style='color:#808030; '>.</span>mallowigi<span style='color:#808030; '>.</span>idea<span style='color:#808030; '>.</span>MTLafComponent<span style='color:#808030; '>.</span>appFrameCreated<span style='color:#808030; '>(</span>MTLafComponent<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>70</span><span style='color:#808030; '>)</span>
at java<span style='color:#808030; '>.</span>base<span style='color:#808030; '>/</span>jdk<span style='color:#808030; '>.</span>internal<span style='color:#808030; '>.</span>reflect<span style='color:#808030; '>.</span>NativeMethodAccessorImpl<span style='color:#808030; '>.</span>invoke0<span style='color:#808030; '>(</span>Native Method<span style='color:#808030; '>)</span>
at java<span style='color:#808030; '>.</span>base<span style='color:#808030; '>/</span>jdk<span style='color:#808030; '>.</span>internal<span style='color:#808030; '>.</span>reflect<span style='color:#808030; '>.</span>NativeMethodAccessorImpl<span style='color:#808030; '>.</span>invoke<span style='color:#808030; '>(</span>NativeMethodAccessorImpl<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>62</span><span style='color:#808030; '>)</span>
at java<span style='color:#808030; '>.</span>base<span style='color:#808030; '>/</span>jdk<span style='color:#808030; '>.</span>internal<span style='color:#808030; '>.</span>reflect<span style='color:#808030; '>.</span>DelegatingMethodAccessorImpl<span style='color:#808030; '>.</span>invoke<span style='color:#808030; '>(</span>DelegatingMethodAccessorImpl<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>43</span><span style='color:#808030; '>)</span>
at java<span style='color:#808030; '>.</span>base<span style='color:#808030; '>/</span>java<span style='color:#808030; '>.</span>lang<span style='color:#808030; '>.</span>reflect<span style='color:#808030; '>.</span>Method<span style='color:#808030; '>.</span>invoke<span style='color:#808030; '>(</span>Method<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>566</span><span style='color:#808030; '>)</span>
at com<span style='color:#808030; '>.</span>intellij<span style='color:#808030; '>.</span>util<span style='color:#808030; '>.</span>messages<span style='color:#808030; '>.</span>impl<span style='color:#808030; '>.</span>MessageBusImpl<span style='color:#808030; '>.</span>invokeListener<span style='color:#808030; '>(</span>MessageBusImpl<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>554</span><span style='color:#808030; '>)</span>
at com<span style='color:#808030; '>.</span>intellij<span style='color:#808030; '>.</span>util<span style='color:#808030; '>.</span>messages<span style='color:#808030; '>.</span>impl<span style='color:#808030; '>.</span>MessageBusConnectionImpl<span style='color:#808030; '>.</span>deliverMessage<span style='color:#808030; '>(</span>MessageBusConnectionImpl<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>139</span><span style='color:#808030; '>)</span>
at com<span style='color:#808030; '>.</span>intellij<span style='color:#808030; '>.</span>util<span style='color:#808030; '>.</span>messages<span style='color:#808030; '>.</span>impl<span style='color:#808030; '>.</span>MessageBusImpl<span style='color:#808030; '>.</span>doPumpMessages<span style='color:#808030; '>(</span>MessageBusImpl<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>472</span><span style='color:#808030; '>)</span>
at com<span style='color:#808030; '>.</span>intellij<span style='color:#808030; '>.</span>util<span style='color:#808030; '>.</span>messages<span style='color:#808030; '>.</span>impl<span style='color:#808030; '>.</span>MessageBusImpl<span style='color:#808030; '>.</span>pumpWaitingBuses<span style='color:#808030; '>(</span>MessageBusImpl<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>432</span><span style='color:#808030; '>)</span>
at com<span style='color:#808030; '>.</span>intellij<span style='color:#808030; '>.</span>util<span style='color:#808030; '>.</span>messages<span style='color:#808030; '>.</span>impl<span style='color:#808030; '>.</span>MessageBusImpl<span style='color:#808030; '>.</span>pumpMessages<span style='color:#808030; '>(</span>MessageBusImpl<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>420</span><span style='color:#808030; '>)</span>
at com<span style='color:#808030; '>.</span>intellij<span style='color:#808030; '>.</span>util<span style='color:#808030; '>.</span>messages<span style='color:#808030; '>.</span>impl<span style='color:#808030; '>.</span>MessageBusImpl<span style='color:#808030; '>.</span>sendMessage<span style='color:#808030; '>(</span>MessageBusImpl<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>404</span><span style='color:#808030; '>)</span>
at com<span style='color:#808030; '>.</span>intellij<span style='color:#808030; '>.</span>util<span style='color:#808030; '>.</span>messages<span style='color:#808030; '>.</span>impl<span style='color:#808030; '>.</span>MessageBusImpl<span style='color:#808030; '>.</span>lambda<span style='color:#ffffff; background:#dd0000; font-weight:bold; font-style:italic; '>$createTopicHandler</span><span style='color:#ffffff; background:#dd0000; font-weight:bold; font-style:italic; '>$3</span><span style='color:#808030; '>(</span>MessageBusImpl<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>243</span><span style='color:#808030; '>)</span>
at com<span style='color:#808030; '>.</span>sun<span style='color:#808030; '>.</span>proxy<span style='color:#808030; '>.</span>$Proxy41<span style='color:#808030; '>.</span>appFrameCreated<span style='color:#808030; '>(</span>Unknown Source<span style='color:#808030; '>)</span>
at com<span style='color:#808030; '>.</span>intellij<span style='color:#808030; '>.</span>idea<span style='color:#808030; '>.</span>IdeStarter<span style='color:#808030; '>.</span><span style='color:#400000; '>main</span><span style='color:#808030; '>(</span>IdeStarter<span style='color:#808030; '>.</span>kt<span style='color:#800080; '>:</span><span style='color:#008c00; '>102</span><span style='color:#808030; '>)</span>
at com<span style='color:#808030; '>.</span>intellij<span style='color:#808030; '>.</span>idea<span style='color:#808030; '>.</span>ApplicationLoader<span style='color:#ffffff; background:#dd0000; font-weight:bold; font-style:italic; '>$startApp</span><span style='color:#ffffff; background:#dd0000; font-weight:bold; font-style:italic; '>$8</span><span style='color:#808030; '>.</span>run<span style='color:#808030; '>(</span>ApplicationLoader<span style='color:#808030; '>.</span>kt<span style='color:#800080; '>:</span><span style='color:#008c00; '>222</span><span style='color:#808030; '>)</span>
at java<span style='color:#808030; '>.</span>base<span style='color:#808030; '>/</span>java<span style='color:#808030; '>.</span>util<span style='color:#808030; '>.</span>concurrent<span style='color:#808030; '>.</span>CompletableFuture<span style='color:#ffffff; background:#dd0000; font-weight:bold; font-style:italic; '>$UniRun</span><span style='color:#808030; '>.</span>tryFire<span style='color:#808030; '>(</span>CompletableFuture<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>783</span><span style='color:#808030; '>)</span>
at java<span style='color:#808030; '>.</span>base<span style='color:#808030; '>/</span>java<span style='color:#808030; '>.</span>util<span style='color:#808030; '>.</span>concurrent<span style='color:#808030; '>.</span>CompletableFuture<span style='color:#808030; '>.</span>postComplete<span style='color:#808030; '>(</span>CompletableFuture<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>506</span><span style='color:#808030; '>)</span>
at java<span style='color:#808030; '>.</span>base<span style='color:#808030; '>/</span>java<span style='color:#808030; '>.</span>util<span style='color:#808030; '>.</span>concurrent<span style='color:#808030; '>.</span>CompletableFuture<span style='color:#808030; '>.</span>postFire<span style='color:#808030; '>(</span>CompletableFuture<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>610</span><span style='color:#808030; '>)</span>
at java<span style='color:#808030; '>.</span>base<span style='color:#808030; '>/</span>java<span style='color:#808030; '>.</span>util<span style='color:#808030; '>.</span>concurrent<span style='color:#808030; '>.</span>CompletableFuture<span style='color:#ffffff; background:#dd0000; font-weight:bold; font-style:italic; '>$UniCompose</span><span style='color:#808030; '>.</span>tryFire<span style='color:#808030; '>(</span>CompletableFuture<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>1085</span><span style='color:#808030; '>)</span>
at java<span style='color:#808030; '>.</span>base<span style='color:#808030; '>/</span>java<span style='color:#808030; '>.</span>util<span style='color:#808030; '>.</span>concurrent<span style='color:#808030; '>.</span>CompletableFuture<span style='color:#ffffff; background:#dd0000; font-weight:bold; font-style:italic; '>$Completion</span><span style='color:#808030; '>.</span>run<span style='color:#808030; '>(</span>CompletableFuture<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>478</span><span style='color:#808030; '>)</span>
at java<span style='color:#808030; '>.</span>base<span style='color:#808030; '>/</span>java<span style='color:#808030; '>.</span>util<span style='color:#808030; '>.</span>concurrent<span style='color:#808030; '>.</span>ThreadPoolExecutor<span style='color:#808030; '>.</span>runWorker<span style='color:#808030; '>(</span>ThreadPoolExecutor<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>1128</span><span style='color:#808030; '>)</span>
at java<span style='color:#808030; '>.</span>base<span style='color:#808030; '>/</span>java<span style='color:#808030; '>.</span>util<span style='color:#808030; '>.</span>concurrent<span style='color:#808030; '>.</span>ThreadPoolExecutor<span style='color:#ffffff; background:#dd0000; font-weight:bold; font-style:italic; '>$Worker</span><span style='color:#808030; '>.</span>run<span style='color:#808030; '>(</span>ThreadPoolExecutor<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>628</span><span style='color:#808030; '>)</span>
at java<span style='color:#808030; '>.</span>base<span style='color:#808030; '>/</span>java<span style='color:#808030; '>.</span>lang<span style='color:#808030; '>.</span>Thread<span style='color:#808030; '>.</span>run<span style='color:#808030; '>(</span>Thread<span style='color:#808030; '>.</span>java<span style='color:#800080; '>:</span><span style='color:#008c00; '>834</span><span style='color:#808030; '>)</span>
</pre>
<!—Created using ToHtml.com on 2020-04-14 03:58:11 UTC —>

**Comment by mallowigi** [comment]
Hi! Are you using a bundled theme perhaps?

**Comment by tarasivashchuk** [comment]
Yup - Solarized Light and Dark

**Comment by mallowigi** [comment]
Ok, I managed to reproduce.

**Comment by tarasivashchuk** [comment]
Great! Let me know if there’s anything else I can help with!

**Comment by timtimmahh** [comment]
Getting this as well on WebStorm 2020.1 although it didn't seem to be a problem with the IDE because I'm using this in CLion 2020.1 Release Candidate and Intellij Ultimate 2020.1 as well with no issues. 
The issue I have mainly in WebStorm is that I can't choose any external themes that I have installed (there's now a theme with the name "External" but it doesn't seem to do anything). I was able to fix that by moving the material-theme-jetbrains-5.1.2.jar I had in my CLion plugins folder to WebStorm and it worked perfectly. After then updating to 5.1.3 from the plugin manager and restarting, that exception occurred and the external themes aren't available again, so it seems to be something with version 5.1.3 of the plugin that has the issue..
